### PR TITLE
Increment template version number for consistency

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

**Review by beseven — APPROVED** [review]
LOTM!

File: templates/c2s-gp-finder/config.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 name: C2S - GP Finder
 description: |
-  Connecting to Services - GP Finder (require separate loadbalancer stack or traefik lb running in the stack)
-version: 1.0.1
+  GP Finder (requires separate load balancer stack or traefik lb running in the stack)
```
Comment (beseven) [inline templates/c2s-gp-finder/config.yml]: @neilbmclaughlin @adamgraves-choices is there a reason the version is gone? do we still need it?

**Review by adamgraves-choices — APPROVED** [review]

File: templates/c2s-gp-finder/config.yml (hunk 1 above)
Comment (adamgraves-choices) [reply templates/c2s-gp-finder/config.yml]: This should be 

version: 1.0.3

**Review by adamgraves-choices — CHANGES_REQUESTED** [review]

**Comment by neilbmclaughlin** [comment]
Because of the frequent force pushes to diagnose template versioning issues this PR is a mess so closing it and reopening another.

**Pull request closed without merging**